src/en/guides/office2016/accessible-pdf-documents.html

```diff
@@ -6,12 +6,12 @@
 
 <div class="row mrgn-tp-lg">
 	<ul class="toc lst-spcd col-md-12">
-		<li class="col-md-4 col-sm-6"><a class="list-group-item" href="../">1. How to create accessible documents</a></li>
-		<li class="col-md-4 col-sm-6"><a class="list-group-item" href="../accessible-word-documents">2. Accessible Word documents</a></li>
-		<li class="col-md-4 col-sm-6 clr-lft-sm"><a class="list-group-item active">3. Accessible PDF documents</a></li>
-		<li class="col-md-4 col-sm-6 clr-lft-md clr-lft-lg"><a class="list-group-item" href="../accessible-powerpoint-documents">4. Accessible PowerPoint documents</a></li>
-		<li class="col-md-4 col-sm-6 clr-lft-sm"><a class="list-group-item" href="../accessible-excel-documents">5. Accessible Excel documents</a></li>
-		<li class="col-md-4 col-sm-6"><a class="list-group-item" href="../accessible-visio-diagrams">6. Accessible Visio diagrams</a></li>
+		<li class="col-md-4 col-sm-6"><a class="list-group-item active">1. How to create accessible documents</a></li>
```

Comment:
The links should stay the same. It looks like you copied this from the index file and pasted it in all the other files. This broke all the links in the TOC navigation on all the other pages.